### ISSUE TITLE
ConfigurationGenerator is not setting ezpublish.system.cache_service_name

### ISSUE THREAD

**Issue by SalvatorePollaci** [netgen/NetgenAdminUIBundle#20]
ConfigurationGenerator(/vendor/netgen/admin-ui-bundle/bundle/Installer/Generator/ConfigurationGenerator.php) is not setting the 'cache_service_name' attribute to '%cache_pool%'  in method generate.

Consequently if NetgenAdminUIBundle is used with Legacy Bridge:

- The Legacy Admin Ui will comunicate with symfony's cache pool 'cache.app'

- Whereas EZ REST API will comunicate with symfony's cache pool 'cache.redis'

So if a content is published from Legacy Admin UI, keys in pool 'cache.app' will be purged, leaving 'cache.redis' keys untouched and EZ REST API will return stale data.

Alternatively state in README.md that such attribute must be set to avoid the above problems.

**Comment by emodric** [comment]
I'd rather it being mentioned in README.md file for two reasons:

1) It's a configuration which does not match all usecases, therefore cannot be autogenerated
2) It is a bug in eZ Platform, so I'd rather avoid writing code that works around a bug (which was recently fixed too)

@SalvatorePollaci Do you want to create a PR? :)

**Comment by SalvatorePollaci** [comment]
Yes sound reasonable to me too. When I have a bit of free time I will create a PR on INSTALL.md
